notebooks/regression/exoplanets.md

```diff
@@ -30,12 +30,12 @@ if install_mapie:
 
 ```python
 import warnings
-from typing_extensions import TypedDict
-from typing import Union
+import matplotlib.pyplot as plt
 import numpy as np
 import pandas as pd
-import matplotlib.pyplot as plt
 import seaborn as sns
+from typing_extensions import TypedDict
+from typing import Union
 from sklearn.compose import ColumnTransformer
```

Comment:
inverser sklearn and typing (ordre alphabétique?)

Reply:
Je commence par les `from` puis par les `import` (voir commentaire de Vincent).